src/main/kotlin/com/zepben/evolve/streaming/grpc/GrpcChannelFactory.kt

```diff
@@ -1,46 +0,0 @@
-/*
- * Copyright 2020 Zeppelin Bend Pty Ltd
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/.
- */
-package com.zepben.evolve.streaming.grpc
-
-import com.zepben.evolve.streaming.get.CimConsumerClient
-import io.grpc.netty.shaded.io.grpc.netty.GrpcSslContexts
-import io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder
-import java.io.File
-
-data class BadConfigException(val msg: String) : Exception(msg)
-
-object GrpcChannelFactory {
-
-    /**
-     * Create a gRPC Channel with the specified [config] that can be passed to a gRPC based [CimConsumerClient] implementation.
-     */
-    @JvmStatic
-    fun create(config: ConnectionConfig): GrpcChannel {
```

Comment:
comment: Typically we wouldn't delete something like this because it's a needless breaking change, but in this case we only have very limited uses of this GrpcChannelFactory so i'm happy to force other repos to update to the new connect methods. However this should be noted as a breaking change in the changelog and ideally have a comment to indicate what you should use instead.